scripts.py

```diff
@@ -411,43 +423,60 @@ def log_missing_identifiers(self, requested_identifiers, included_qu):
             if not work:
                 detail_list += (bullet + "%r : No Work found." % license_pool)
                 continue
+            detail_list += (bullet + "%r : Unknown error.")
```

Comment:
I don't know if this will ever happen, but it's supposed to reference the identifier. It's fixed in the latest commit.